diff -urbB kernel.unpatched/debian.master/config/config.common.ubuntu kernel/debian.master/config/config.common.ubuntu
--- kernel.unpatched/debian.master/config/config.common.ubuntu	2022-01-21 13:32:27.000000000 +0100
+++ kernel/debian.master/config/config.common.ubuntu	2022-02-22 22:02:22.000000000 +0100
@@ -1273,7 +1273,7 @@
 CONFIG_BOUNCE=y
 CONFIG_BPF=y
 CONFIG_BPFILTER=y
-CONFIG_BPFILTER_UMH=m
+CONFIG_BPFILTER_UMH=y
 CONFIG_BPF_EVENTS=y
 CONFIG_BPF_JIT=y
 CONFIG_BPF_JIT_ALWAYS_ON=y
