diff -urbB kernel.unpatched/debian.master/config/config.common.ubuntu kernel/debian.master/config/config.common.ubuntu
--- kernel.unpatched/debian.master/config/config.common.ubuntu	2022-01-21 13:32:27.000000000 +0100
+++ kernel/debian.master/config/config.common.ubuntu	2022-02-22 22:02:22.000000000 +0100
@@ -4164,7 +4164,8 @@
 CONFIG_HAVE_FUNCTION_GRAPH_TRACER=y
 CONFIG_HAVE_FUNCTION_TRACER=y
 CONFIG_HAVE_FUTEX_CMPXCHG=y
-CONFIG_HAVE_GCC_PLUGINS=y
+CONFIG_HAVE_GCC_PLUGINS=n
+CONFIG_GCC_PLUGINS=n
 CONFIG_HAVE_GENERIC_VDSO=y
 CONFIG_HAVE_HARDENED_USERCOPY_ALLOCATOR=y
 CONFIG_HAVE_HARDLOCKUP_DETECTOR_ARCH=y

